spec/helpers/attachments_helper_spec.rb

```diff
@@ -20,5 +20,15 @@
         expect(attachments_with_size(attachments)).to eq []
       end
     end
+
+    context 'without original_filename' do
+      let(:attachment1) { create :attachment, original_filename: nil }
+      let(:attachment2) { create :attachment, original_filename: 'fake name' }
+      let(:attachments) { [attachment1, attachment2] }
+
+      it 'returns attachment_type name in its place' do
+        expect(attachments_with_size(attachments)).to match ['statement_of_case (15.7 KB)', 'fake name (15.7 KB)']
```

Comment:
is there any chance these attachments could be created in a different order and therefore the test would fail? If so we could use the `match_array` option